gensrc/proto/olap_file.proto

```diff
@@ -287,6 +287,7 @@ message TabletMetaPB {
     optional RowsetTypePB preferred_rowset_type = 16;
     optional TabletTypePB tablet_type = 17;
     optional TabletUpdatesPB updates = 50; // used for new updatable tablet
+    optional bool use_persistent_index = 51 [default = false]; // used for persistent index in primary index
```

Comment:
If the old table wants to enable the persistent index, how to operate?

Reply:
> If the old table wants to enable the persistent index, how to operate?

Actually it is not allowed so far. This feature is only available for newly created PrimaryKey tables until full testing is performed. 

When the feature is stable, we can enable persistent index for all PrimaryKey tables and use a config in be.conf to decide enable or not.